guardian-ui/README.md

```diff
@@ -0,0 +1,33 @@
+# Guardian UI
+
+## Prerequisites NPM and Yarn
+
+[NPM](https://www.npmjs.com/) - (use [nvm](https://github.com/nvm-sh/nvm))
+
+[Yarn](https://yarnpkg.com/getting-started/installhttps://classic.yarnpkg.com/lang/en/docs/install/) - `npm install --global yarn`
```

Comment:
This link seems to be dead, but that might become irrelevant once @okjodom's concerns are addressed

Reply:
@garyray-k let's remove this **Prerequisites** section and instead, start right away with the instructions you have listed in "Run the UI".

The goal here is to have a standard dev environment powered by nix. As such it's not a prerequisite to install npm / yarn on your dev machine. Just run the nix shell and you are good to go

Reply:
Done.